storybook/preview.js

```diff
@@ -1,6 +1,5 @@
-import { addParameters, configure } from '@storybook/react';
+import { configure } from '@storybook/react';
```

Comment:
where is the theme defined now that `addParameters` got removed?

Reply:
That theme wasnt being used!
the only theme used on stories was this one: `src/js/all/stories/theme.js`

Reply:
Nice clean up.